src/Emogrifier/CssConcatenator.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Pelago\Emogrifier;
+
+/**
+ * Facilitates building a CSS string by appending rule blocks one at a time, checking whether the media query,
+ * selectors, or declarations block are the same as those from the preceding block and combining blocks in such cases.
```

Comment:
A usage example would be great here in the class doc.

Reply:
Added